pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "API wrapper and utils for QuantAQ, Inc."
 authors = ["David H Hagan <david.hagan@quant-aq.com>"]
 license = "Apache License 2.0"
 readme = "README.md"
-homepage = ""
+homepage = "https://quant-aq.github.io/py-quantaq"
```

Comment:
Had to put something here to get tests to run--is this correct?

Reply:
huh. I wonder when that requirement changed!